shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
@Balletie @gfokkema Please discuss. I don't think it belongs here as this class is more of a library kind and should therefore not control the ui. It's better to have the `IEditor` and `IDisplay` implementations set defaults.